clients/src/main/java/org/apache/kafka/common/requests/LeaderAndIsrResponse.java

```diff
@@ -58,8 +68,17 @@ public Errors error() {
         Errors error = error();
         if (error != Errors.NONE)
```

Comment:
nit: We should add braces here since there are multiple lines inside the `if` statement